tools/SynchronizeData.py

```diff
@@ -0,0 +1,607 @@
+"""
+Created July 11th, 2019
+
+@author: zackorenberg
+
+This widget is meant to be used both internally/externally to synchronize independent datafiles. It will treat each data file as a set, and therefore support elementary set operations
+"""
```

Comment:
It would be useful to describe why such a file is useful by giving a use case example and how this will solve it